### ISSUE TITLE
Angular won't bootstrap using this example

### ISSUE THREAD

**Issue by mrsquish** [Hongbo-Miao/office-addin-angular#2]
Hey Hongbo, 

So just now I started from scratch so that I had accurate reproducible steps:

Cloned Down the Repository. 
Ran npm install
Copied the manifest to my existing shared folder
Ran ng serve
Loaded up Excel -> Insert -> My Plugins, selected office-addin-angular plugin, pressed ok
Home -> show taskpane. 
The addin loads, but essentially I just get the index.html output (Loading...). 

I finally used the F12 Debugger in Windows10 today to get some debug output. And I get an error 👍 

SCRIPT5022: Office.js has not been fully loaded yet. Please try again later or make sure to add your initialization code on the Office.initialize function.
polyfills.bundle.js (2639,25)

**Comment by mrsquish** [comment]
Sorry Hongbo, before the above error I got this 👍 

SCRIPT5007: Unable to get property 'apply' of undefined or null reference
vendor.bundle.js (25195,5)

**Comment by mrsquish** [comment]
Hey Hongbo, 

I found the issue. Basically, out of the box Angular CLI doesn't support IE browsers. Which meant that it didn't include the polyfills required to allow the .assign function to work in IE11. 

see : [This stackoverflow page](https://stackoverflow.com/questions/41276692/angular2-ie11-unable-to-get-property-apply-of-undefined-or-null-reference)

I just needed to include the polyfills in the polyfills.ts and everything worked.

FYI, I tried pushing in the changes but can't.
